README.md

```diff
@@ -252,7 +252,7 @@ will stop the build.
 Run the tests with the following command:
 
 ```sh
-docker build -t git-resource .
+docker build -t git-resource -f ./dockerfiles/ubuntu/Dockerfile .
```

Comment:
Hey, as we're not dropping the support for `alpine` in this PR, I'd say we should advertise the fact that it's possible to build and test both.

Wdyt?

thanks!

Reply:
I would assume that since the path is provided as `dockerfiles/.../Dockerfile` it would be intuitive enough to check that directory and see that there are multiple options.

Although now that I think about it, the README should use `alpine` as that's still the default option for now

Reply:
Sounds good 👍